site/en/blog/new-in-chrome-93/index.md

```diff
@@ -0,0 +1,253 @@
+---
+title: New in Chrome 93
+description: >
+  Chrome 93 is rolling out now! You can now load CSS style sheets with import
+  statements, just like JavaScript modules. Installed PWAs can register as
+  URL handlers, making it possible for users to jump straight into your PWA.
+  The Multi-Screen Window Placement API has been updated based on your
+  feedback, and starts a second origin trial. We're shortening the release
+  cycle for stable to 4-weeks. And there's plenty more.
+layout: 'layouts/blog-post.njk'
+date: 2021-08-31
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/WHMxxltpvNS4t3YAEkLL.png'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-93
+---
+
+{# YouTube id='COMING_SOON' #}
+
+Here's what you need to know:
+
+* You can now load CSS style sheets with [`import` statements](#css-modules),
+  just like JavaScript modules.
+* Installed PWAs can register as [URL handlers](#url-handlers), making it
+  possible for users to jump straight into your PWA.
+* The [Multi-Screen Window Placement API](#window-placement) has been updated
+  based on your feedback, and starts a second origin trial.
+* The [PWA Summit](#pwa-summit) is coming up October 6-7.
+* And there's plenty [more](#more).
+
+I'm [Pete LePage](https://petelepage.com), working, and shooting
+from home, let's dive in and see what's new for developers in Chrome 93!
+
+## CSS Module Scripts {: #css-modules }
+
+You can now load CSS style sheets with `import` statements, just like
+JavaScript modules. The style sheets can then be applied to the document or
+shadow roots in the same manner as construct-able stylesheets.
+
+The new CSS Module Scripts feature is great for custom elements. And unlike
+other ways of applying CSS from JavaScript, there is no need to create
+elements, or mess with Java Script strings of CSS text.
+
+To use it, import the style sheet with `assert {type: 'css'}`, then apply it
+to the `document` or `shadowRoot` by calling `adoptedStyleSheets`.
+
+```js
+import sheet from './styles.css' assert { type: 'css' };
+document.adoptedStyleSheets = [sheet];
+shadowRoot.adoptedStyleSheets = [sheet];
+```
+
+But beware, if you leave off the `assert` - the file will be treated as
+JavaScript, and won't work!
+
+Check out [Using CSS Module Scripts to import stylesheets][wd-css-modules] on
+web.dev for complete details.
+
+## Multi-Screen Window Placement API {: #window-placement }
+
+For some apps, opening new windows, and putting them in specific places,
+or specific displays is an important feature. For example, when using Slides to
```

Comment:
```suggestion
For some apps, opening new windows and putting them in specific places
or specific displays is an important feature. For example, when using Slides to
```

Reply:
Is 'Slides' and app name? If not, it should be lower case.

Reply:
Slides is the name of our slides/presentation app. I think.